deployment/v3/utils/prop_migrator/README.md

```diff
@@ -0,0 +1,225 @@
+
+| Old Property File | Key  | Old Value |
+|-------------------|------|-----------|
+| file1.properties  | key1 | value1    |
+| file2.properties  | key2 | value2    |
+| ...               | ...  | ...       |
+
+### latest_file_only.csv
+
+This file contains property combinations that are present only in the latest property files. It has the following structure:
+
+| Latest Property File | Key  | Latest Value |
+|----------------------|------|--------------|
+| file1.properties     | key1 | value1       |
+| file2.properties     | key2 | value2       |
+| ...                  | ...  | ...          |
+
+### intentionally_removed_in_lts.csv
+
+This file contains property combinations that were intentionally removed in the latest version. It has the following structure:
+
+| Property File Name | Key  |
+|--------------------|------|
+| file1.properties   | key1 |
+| file2.properties   | key2 |
+| ...                | ...  |
+
+### decent_default_value.csv
+
+This file contains information about decent default property values for the latest property files. It has the following structure:
+
+| Property File Name | Key  | Latest Value |
+|--------------------|------|--------------|
+| file1.properties   | key1 | value1       |
+| file2.properties   | key2 | value2       |
+| ...                | ...  | ...          |
+
+### manual-configuration.csv
+
+This file is created or updated by the script and includes unresolved combinations and combinations that require further manual configuration. It has the following structure:
+
+| Property File Name | Key  | Old Value | Latest Value | Configuration Action |
+|--------------------|------|-----------|--------------|---------------------|
+| file1.properties   | key1 | value1    | value2       | update-if-required  |
+| file2.properties   | key2 | value3    | value4       | copy-property       |
+| ...                | ...  | ...       | ...          | ...                 |
+
+## Output
+
+The Property File Updater generates the following output:
+
+### Updated Property Files
+
+This section lists the property files that were updated during the process. It includes the names of the files and the keys that were modified.
+
+### Logs
+
+The script generates a log file (`logs.txt`) that provides detailed information about the update process. It includes information about matching combinations, unresolved combinations, similar latest values, and more. The log file helps you understand the changes made to the property files and allows you to review the update process.
+
+### Manual Configuration CSV
+
+The script creates or updates the `manual-configuration.csv` file. This file includes unresolved combinations and combinations that require further manual configuration. You can use this file to manually update the property files based on the required configuration actions mentioned in the file.
+
+## Notes
```

Comment:
We do not need to put ## for Notes .. we can just add it like this _Note_:

Reply:
updated